package.json

```diff
@@ -8,7 +8,7 @@
     "test": "mocha ./spec/index.js",
     "types": "tsc",
     "clean": "rm -rf lib && mkdir lib && rm -fr types && mkdir types",
-    "build": "babel src --out-dir lib --copy-files --extensions '.ts,.js,.jsx,.tsx'",
+    "build": "babel src --out-dir lib --copy-files --extensions .ts,.js,.jsx,.tsx",
```

Comment:
removing `'` will break the command for unix machines, you can try to change it to:
```
    "build": "babel src --out-dir lib --copy-files --extensions \".ts,.js,.jsx,.tsx\"",
```
otherwise I would just keep the old version